apps/jenkins/jenkins/jenkins.yaml

```diff
@@ -112,8 +112,8 @@ spec:
               securityRealm:
                 azure:
                   cacheDuration: 3600
-                  clientId: "${cft-jenkins-sp-client-id}"
-                  clientSecret: "${cft-jenkins-sp-secret}"
+                  clientId: "900f24bb-03e4-4eb2-b400-3e0f96e1cbbe"
```

Comment:
this needs to be in vault doesn't it because this config is in the base?

Reply:
[8d029c6](https://github.com/hmcts/cnp-flux-config/pull/25813/commits/8d029c6e3b449976f7bdcee5f00500c59f04d43e)